### PR TITLE
add new sample DIY mosaic visualization notebook

### PR DESCRIPTION
**Pull request by emmalu**
## Summary:
**What** did I do?
Followed [this updated documentation](https://docs.maap-project.org/en/develop/technical_tutorials/visualization/visualizing_mosaicjson.html) to create a sample notebook that covers how to create a mosaic from an array of COG raster urls, then load that mosaic as a Tile layer within a stac_ipyleaflet map instance.

**Why** *** did I do it? (What was the motivation?)
- To help users extend current stac_ipyleaflet functionality.

**How** *** can someone test or demo my changes?
- Follow the notebook. :)

## Checklist before requesting a review:
- [ ] My code follows the guidelines of this project
- [X] I  performed a self-review of my code changes
- [ ] I have added my changes to the change log (`HISTORY.rst`)
- [X] I have completed spell-checking, removed unnecessary print statements & commented-out code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/sample_notebooks/visualize-mosaic.ipynb [hunk 1]
```diff
@@ -0,0 +1,429 @@
+{
```
Comment (jjfrench) [inline docs/sample_notebooks/visualize-mosaic.ipynb:1]: <p>if it's MAAP specific, should it go under MAAP documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/stac_ipyleaflet/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (wildintellect) [reply docs/sample_notebooks/visualize-mosaic.ipynb:1]: Valid point, makes me wonder if we just need to find a more publicly available mosaic example for these docs. Agreed this specific example should move to MAAP docs. cc: @grallewellyn

Comment (grallewellyn) [reply docs/sample_notebooks/visualize-mosaic.ipynb:1]: I was working on replacing mentions of ipycmc in the docs with stac_ipyleaflet, so I will make sure this gets added to maap-documentation. Thank you so much!

Comment (grallewellyn) [reply docs/sample_notebooks/visualize-mosaic.ipynb:1]: https://github.com/MAAP-Project/maap-documentation/pull/333

Comment (emmalu) [reply docs/sample_notebooks/visualize-mosaic.ipynb:1]: > Valid point, makes me wonder if we just need to find a more publicly available mosaic example for these docs.

That would be great - and we could use one of the URLs in the `raster` DIY notebook too, which is also MAAP-specific, I realize now.


And we also probably need to remove the blocks about deriving the asset path from bucket variable and environment variables, which are relative to the environment...

**Comment by emmalu** [comment]
Closing this PR in favor of Llewellyn's MAAP documentation updates that include [visualizing mosaic raster using stac_ipyeaflet](https://docs.maap-project.org/en/latest/technical_tutorials/visualization/visualize_mosaic_stac_ipyleaflet.html)

**Pull request closed without merging**